### ISSUE TITLE
Button to hide pokemons already sniped

### ISSUE THREAD

**Issue by supertx2** [5andr0/PogoLocationFeeder#333]
Can you guys add a button to hide pokemons already sniped in the grid?

**Comment by tensei** [comment]
delete after snipe button clicked?

**Comment by supertx2** [comment]
yes, something like that, or a hide button like in www.rarespawns.be this wold be very helpfull  to keep snipes organized ^^
